### PR TITLE
framework/src/arastorage: fix multiple definition of `g_storage_write_buffer' error

### PR DESCRIPTION
**Pull request by abhishek-samsung**
Multiple definition of `g_storage_write_buffer' error occurs when using docker 1.6.0
g_storage_write_buffer was only used in storage_interface.c file. So moved the
definition from the header into the file

Signed-off-by: Abhishek Akkabathula <a.akkabathul@samsung.com>

### CODE REVIEW

File: framework/src/arastorage/storage_interface.c [hunk 1]
```diff
@@ -84,9 +84,14 @@ struct index_record_s {
 };
 
 /****************************************************************************
-* Public Functions
+* Public Variables
 ****************************************************************************/
 #ifdef CONFIG_ARASTORAGE_ENABLE_WRITE_BUFFER
+struct insert_buffer_s g_storage_write_buffer;
```
Comment (jeongchanKim) [inline framework/src/arastorage/storage_interface.c:90]: If this variable `g_storage_write_buffer` uses only in this file, how about adding `static`?

File: framework/src/arastorage/storage_interface.c [hunk 2]
```diff
@@ -84,9 +84,14 @@ struct index_record_s {
 };
 
 /****************************************************************************
-* Public Functions
+* Public Variables
 ****************************************************************************/
 #ifdef CONFIG_ARASTORAGE_ENABLE_WRITE_BUFFER
+static struct insert_buffer_s g_storage_write_buffer;
```
Comment (sunghan-chang) [inline framework/src/arastorage/storage_interface.c:90]: It's `static` global variable. Then, is it `public` or `private` variable?

Comment (thapav) [reply framework/src/arastorage/storage_interface.c:90]: I think this will come under private data

Comment (abhishek-samsung) [reply framework/src/arastorage/storage_interface.c:90]: Forgot to change the heading after adding static. Thanks for pointing it out Mr. Chang. Modified.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**